docs/Dialects/LTL.md

```diff
@@ -167,8 +174,240 @@ property p1; eventually p0; endproperty
 ```
 In this example, `p1` refers to property `p0`, which is illegal in SVA since `p0` itself defines a disable condition.
 
-In contrast, the LTL dialect explicitly allows for properties to be disabled at arbitrary points, and disabled properties to be used in other properties. Since a disabled nested property also disables the parent property, the IR can always be rewritten into a form where there is only one `disable iff` condition at the root of a property expression.
+In contrast, the LTL dialect explicitly allows for properties to be disabled at arbitrary points, and disabled properties to be used in other properties. Since a disabled nested property also disables the parent property, the IR can always be rewritten into a form where there is only one `disable iff` condition at the root of a property expression.  
+  
+## Mapping SVA to CIRCT dialects  
+  
+Knowing how to map SVA constructs to CIRCT is important to allow these to expressed correctly in any front-end. Here you will find a non-exhaustive list of conversions from SVA to CIRCT dialects.  
+
+- **properties**: `!ltl.property`.    
+- **sequences**: `!ltl.sequence`.  
+  
+- **`disable iff (cond)`**:  `ltl.disable %prop if %cond`
+- **local variables**: Not currently possible to encode.  
+
+- **`$rose(a)`**: 
+```mlir
+%1 = ltl.compreg %a, %clock : i1
+%rose = comb.icmp bin ult %1, %a : i1  
+```  
+
+- **`$fell(a)`**: 
+```mlir
+%1 = ltl.compreg %a, %clock : i1
+%fell = comb.icmp bin ugt %a, %1 : i1
+```    
+
+- **`$stable(a)`**:
+```mlir
+%1 = ltl.compreg %a, %clock : i1
+%rose = comb.icmp bin eq %a, %1 : i1
+``` 
+
+- **`$past(a, n)`**: 
+```mlir
+%zero = hw.constant 0 : i1
+%true = hw.constant 1 : i1
+%1 = seq.shitreg n, %a, %clk, %true, powerOn %zero : i1
+``` 
+
+> The following functions should be encoded in the front-ends directly:  
+> - **`$onehot(a)`**: `a.reduce(_+_) == 1`
+> - **`$onehot0(a)`**: `a.reduce(_+_) <= 1`
+> - **`$isunknown(a)`**: `a == X || a == Z`
```

Comment:
Frontend(chisel) cannot represent Z/X? Maybe we need to add an intrinsic here?

Reply:
Maybe nut for now they aren't supported, I guess I'll just write that then

Reply:
Maybe X/Z should have clearer support in the front-ends

Reply:
Agree. That might be a non trivial work we should think carefully.

Reply:
Yes I'm afraid that this kind of approach could bloat the intrinsics available in chisel. I would rather use easily identifiable patterns that we can then identify in exportVerilog to emit these functions only in SV and not have the IR behave as an AST for SVA.

Reply:
FWIW there is an intrinsic for X value. https://github.com/llvm/circt/blob/c76eb120d675fc093ca114d288ca25548be85e7a/include/circt/Dialect/FIRRTL/FIRRTLIntrinsics.td#L175-L189